### ISSUE TITLE
Better exception handling in case of syntax errors

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#96]
```
What steps will reproduce the problem?
command: "sms:Contactname:"

What is the expected output?
Syntax Error

What do you see instead?
Error: java.lang.IllegalArgumentException: Invalid message body

What version of the product are you using?
Sources
```

Original issue reported on code.google.com by `ThDietri...@gmail.com` on 10 Feb 2011 at 7:15

**Comment by GoogleCodeExporter** [comment]
```
We could throw an invalid syntax exception instead, which contains a 
description for the syntax and send this string(s) back instead of the plain 
exception string.
```

Original comment by `fschm...@gmail.com` on 10 Feb 2011 at 7:41
- Changed state: **Accepted**
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 10 Feb 2011 at 9:42
- Changed title: **Better exception handling in case of syntax errors**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 10 Feb 2011 at 9:42
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 12 Feb 2011 at 3:59
- Added labels: **Type-Defect**
- Removed labels: **Type-Enhancement**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 12 Feb 2011 at 4:18
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.9.1
```

Original comment by `Florent....@gmail.com` on 13 Feb 2011 at 12:48
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
